### PR TITLE
VIITE-1397 optimize road address queries

### PR DESCRIPTION
**Pull request by skosonen**
Combine LRM_POSITION table to ROAD_ADDRESS and PROJECT_LINK and PROJECT_LINK_HISTORY -tables.
VIITE-1397, VIITE-1426

### CODE REVIEW

**Comment by skosonen** [comment]
This is still work in progress.

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala [hunk 1]
```diff
@@ -274,6 +270,7 @@ object RoadAddressDAO {
     }
   }
 
+  // TODO REMOVE?
```
Comment (leirasf) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala]: This implicit can not be removed. Is used in all the `queryNA[RoadAddress]`.

**Comment by skosonen** [comment]
All tests pass now. (Except those two ignored ones that were using "Kauniainen" LRM_POSITIONs)

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala [hunk 2]
```diff
@@ -869,6 +853,7 @@ object RoadAddressDAO {
         WHERE id = ${roadAddress.id}""".execute
   }
 
+  // TODO
```
Comment (skosonen) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala]: Remove TODO.

**Comment by Campanha** [comment]
Should the ConversionRoadAddress, IncomingRoadAddress and IncomingLrmPosition also be refactored?

**Comment by skosonen** [comment]
@Campanha Those could be also refactored. I just did not want to touch those parts since importing road addresses work now.

**Comment by skosonen** [comment]
All tests pass in Jenkins. Check build #1057.

**Pull request merged**